_plugins/elixir_school_redirects.rb

```diff
@@ -0,0 +1,36 @@
+module ElixirSchoolRedirects
+
+  REDIRECTS = {
```

Comment:
Maybe it makes sense to store this inside a `config` or `_data` file? When we'll need to add or change something, it could probably be more straight-forward to do it there without having to dive into plugin itself.

Reply:
Ha!  I had a comment here that I deleted in which I mentioned my reasoning for this was because I don't foresee any additional changes.  It seemed overkill but I'm happy to do it for conventions sake.